src/gui/tray/usermodel.cpp

```diff
@@ -725,6 +726,12 @@ void User::removeAccount() const
     AccountManager::instance()->save();
 }
 
+void User::slotSendChatMessage(const QString &token, const QString &message, const QString &replyTo){
```

Comment:
@camilasan `QString &replyTo){` missing space between `)` and `{`, also, we don't use JavaScript code style, so, `{` should be on a new line.